### DISCUSSION TITLE
Send JSON to MQTT with Tasmota not working

### DISCUSSION THREAD

**Discussion by raduigret** [arendst/Tasmota#21477] [Q&A] [answered]
I have 2 Tasmotas flashed with same standard FW.
On tasmota A, if I send JSON format via UART, it is send via MQTT as below:
**MQT: tele/tasmota_234e33h/RESULT = {"SerialReceived":{"test":2}}**
but on tasmota B, if I send same payload, it reported as below in console:
**CMD: {"test":2}**

Both have same configuration: 
![image](https://github.com/arendst/Tasmota/assets/80286831/5783e934-c1c6-414b-8f2a-575f35885928)

the only exception is that A (that works) was configured some months before and I forgot what default settings I changed and runs Tasmota 13.4.0.4,
while B (that does not work) has Tasmota 14.0.0 and default settings.

Please help me how to configure it to send received json from serial via MQTT as above.

**Comment by barbudor** [comment, marked as answer]
If I remember you need to send at least 1 char on the serial out to force the serial bridge mode
Otherwise you are still in console mode and what is received is interpreted as a command

**Reply by sfromis** [reply]
Additionally, I'd prefer to actually using serial bridge, meaning configuring pins as `SerBr Tx/Rx` instead of `Serial Tx/Rx`.

**Reply by SkowronK** [reply]
It reminds me my case described 3 hours ago (#21476).
Try downgrading tasmota back to 13.4. I bet it will work fine.

**Reply by arendst** [reply]
@SkowronK you have a total different issue. In that issue the answer is already given. Follow the provided information.

**Comment by raduigret** [comment]
Yes, that was the trick, it works now. Thank you!
